admin/class-acc_user_importer-admin.php

```diff
@@ -635,7 +695,9 @@ public function enqueue_scripts() {
 	}
 	
 	public function log_local_output( $v ) {
```

Comment:
Generally speaking, do we want to have a separate log file for each run? If so, I am wondering if we could leverage the access token we receive from the ACC API plus the start time of the run to have a 'known' log file name. That way we would not need any logic to check if a file exists or not, we just know the file name for the run and that's it.

Of course, this would require that we'd have to pass that information down to the backend for frontend initiated runs. That said, I'd not recommend doing this refactoring in this PR but in a future change.

Reply:
I agree with your proposal, it would be simpler and better. Later.